### DISCUSSION TITLE
winget Failed when searching source; results will not be included: winget No installed package found matching input criteria.

### DISCUSSION THREAD

**Discussion by Lefty4444** [Weatherlights/Winget-AutoUpdate-Intune#23] [General]
Hello! 

We have just deployed this to a couple of machines. My test machine is a vm running Windows 11 ARM using Mac myself with Apple Silicon so my VM is running in Parallells. All installed well, I can check logs but it does not find any updates. I have installed old versions of 7-zip and notepad++. 

It seems my vm can access internet just fine (it did enroll fine into Intune etc) but it cannot update winget list. 

Any ideas?

Thanks

```
winget source list

Name    Argument
-----------------------------------------------------
msstore https://storeedgefd.dsx.mp.microsoft.com/v9.0
winget  https://cdn.winget.microsoft.com/cache
```

```
winget update
Failed in attempting to update the source: winget
Failed when searching source; results will not be included: winget
No installed package found matching input criteria.
```

**Comment by Weatherlights** [comment]
It could be that the ARM version of 7-Zip is not available with Winget.
